src/magnifik.js

```diff
@@ -1,7 +1,13 @@
 (function() {
 
-var Mobify = window.Mobify = window.Mobify || {}
-  , $ = Mobify.$ = Mobify.$ || window.$ || window.Zepto || window.jQuery;
+var Adaptive = window.Adaptive;
+var Mobify = window.Mobify = window.Mobify || {};
+
+if (Adaptive.$ === undefined) {
```

Comment:
I think this is going to fail for projects where `window.Adaptive` does not exist.